config/packages/sylius_twig_hooks.yaml

```
@@ -0,0 +1,31 @@
+sylius_twig_hooks:
+    hooks:
+        'sylius_shop.base.offcanvas':
+            info_box:
+                template: "blocks/info_box.html.twig"
```

This `blocks` directory seems no to be needed here and below 🤔 
```suggestion
                template: "info_box.html.twig"
```